File: linear_algebra/Python/NQM_BT10.py
print("Nhập các phần tử của ma trận 2x2 (a b c d):")
a, b, c, d = map(float, input().split())

det = a * d - b * c

print("Ma trận A:")
print(f"{a}\t{b}")
print(f"{c}\t{d}")

print(f"\nĐịnh thức (det A) = {det}")

File: linear_algebra/C++/NQM_BT10.cpp
#include <iostream>
using namespace std;

int main() {
    double a, b, c, d;
    cout << "Nhap cac phan tu cua ma tran 2x2 (a b c d): ";
    cin >> a >> b >> c >> d;

    double det = a * d - b * c;

    cout << "Ma tran A: " << endl;
    cout << a << "\t" << b << endl;
    cout << c << "\t" << d << endl;

    cout << "Dinh thuc (det A) = " << det << endl;
    return 0;
}
